roles/cifmw_cephadm/tasks/rgw.yml

```diff
@@ -61,14 +61,35 @@
         rgw_frontend_cert: "{{ slurp_cert['content'] | b64decode ~
                                slurp_key['content'] | b64decode }}"
 
+- name: Create default RGW realm
+  block:
+    - name: Get radosgw-admin ceph_cli
+      ansible.builtin.include_tasks: ceph_cli.yml
+      vars:
+        ceph_command: radosgw-admin
+
+    - name: Get default realm
+      become: true
+      ignore_errors: true
+      ansible.builtin.command: |
+        {{ cifmw_cephadm_ceph_cli }} realm get --rgw-realm={{ cifmw_ceph_rgw_realm }}
+      register: rgw_realm
+
+    - name: Create default RGW realm
+      when: rgw_realm.rc > 0
+      ansible.builtin.command: |
+        {{ cifmw_cephadm_ceph_cli }} realm create --rgw-realm={{ cifmw_ceph_rgw_realm }}
```

Comment:
I know nothing about thisare , but when I googled I see there's a "--default" option you can specify, and it's unclear whether the option is relevant when there are multiple realms or just one. I'm only noting it in case it warrants investigation.

Reply:
Thank you, I can try that option and see if anything change (or allows to write less code :D).
From a test I did on an existing `Ceph` cluster, I saw that `realm` was an empty list, and considering we do not deploy rgw multi site in CI, my idea was to create in advance a `default` realm that can match what we already have in the previous versions. It's a parameter, in case we would like to have a better name rather than "default".